src/rust.rs

```diff
@@ -24,10 +29,12 @@ pub fn render(spec: &ast::Spec) -> TokenStream {
 /// Render a struct definition.
 fn render_struct_def(sdef: &ast::StructDef) -> TokenStream {
     let ident = fmt_ident(&sdef.name);
+    let doc_comment = fmt_opt_string(&sdef.doc_comment);
     let fields: Vec<_> = sdef.fields.iter().map(render_pub_field_node).collect();
 
     quote!(
         #[derive(Debug, Clone, serde::Deserialize, serde::Serialize)]
+        #[doc = #doc_comment]
```

Comment:
this should only be rendered if `!doc_comment.is_empty()`.

Reply:
I don't think this is a blocking issue for the generated code. Turning this into an issue.